### PR TITLE
[12.0] queue_job: ease testing

### PR DESCRIPTION
**Pull request by simahawk**
Improves #111 but I need this on v12 now, then I'll update the old one w/ backward port.
/cc @lmignon

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
GREAT! Thank you @simahawk !

File: queue_job/readme/USAGE.rst [hunk 1]
```diff
@@ -1,3 +1,33 @@
 To use this module, you need to:
 
 #. Go to ``Job Queue`` menu
+
+Developers
+----------
```
Comment (pedrobaeza) [inline queue_job/readme/USAGE.rst]: You should use `~~~~` as underlining.

File: queue_job/readme/USAGE.rst [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(
+            cls.env.context,
+            test_queue_job_no_delay=True,  # no jobs thanks
+        ))
+
+Then all your tests won't have any job running.
```
Comment (guewen) [inline queue_job/readme/USAGE.rst]: This sentence is misleading: the job won't run but the methods will.
```suggestion
Then all your tests execute the job methods synchronously without delaying any jobs.
```

**Review by guewen — APPROVED** [review]
Great!

**Comment by simahawk** [comment]
@pedrobaeza @guewen updated.

**Pull request merged**